tests/Rule/LatteTemplatesRule/PresenterWithoutModule/LatteTemplatesRuleForPresenterTest.php

```diff
@@ -498,17 +498,17 @@ public function testRecursion(): void
                 'recursion.latte',
             ],
             [
-                'Dumped type: 9|10',
```

Comment:
It was fixed by above replacement of array_merge with VariablesHelper::merge. Original version mixed all variable in single array instead of replacing merging them and later they were unionised when outputed to template. Original behaviour was wrong (variable remembered all previous values in include-chain) but I was not sure why. Now I found it and fixed it.